src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/SerializableUnimportedExtensionMethods.cs

```diff
@@ -20,14 +19,11 @@ internal sealed class SerializableUnimportedExtensionMethods(
     public readonly ImmutableArray<SerializableImportCompletionItem> CompletionItems = completionItems;
 
     [DataMember(Order = 1)]
-    public readonly bool IsPartialResult = isPartialResult;
-
-    [DataMember(Order = 2)]
     public TimeSpan GetSymbolsTime { get; set; } = getSymbolsTime;
 
-    [DataMember(Order = 3)]
+    [DataMember(Order = 2)]
     public readonly TimeSpan CreateItemsTime = createItemsTime;
 
-    [DataMember(Order = 4)]
+    [DataMember(Order = 3)]
     public readonly TimeSpan? RemoteAssetSyncTime = remoteAssetSyncTime;
```

Comment:
I don't think we need those perf number anymore, at least I haven't looked at them for a long time. Feel free to remove them all from telemetry

Reply:
done.